src/main/java/com/example/bankservice/service/impl/TransactionServiceImpl.java

```diff
@@ -0,0 +1,51 @@
+package com.example.bankservice.service.impl;
+
+import com.example.bankservice.model.Account;
+import com.example.bankservice.model.Transaction;
+import com.example.bankservice.repository.TransactionRepository;
+import com.example.bankservice.service.AccountService;
+import com.example.bankservice.service.TransactionService;
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class TransactionServiceImpl implements TransactionService {
+    private final TransactionRepository transactionRepository;
+    private final AccountService accountService;
+
+    @Override
+    public void save(Account accountTo, Account accountFrom, double amount) {
+        Transaction transactionFrom = new Transaction();
+        transactionFrom.setDate(LocalDateTime.now());
+        transactionFrom.setType(Transaction.Type.OUTCOMING);
+        transactionFrom.setAccountFrom(accountFrom);
+        transactionFrom.setAccountTo(accountTo);
+        transactionFrom.setAmount(amount);
+        System.out.println(accountFrom.getBalance());
+        accountFrom.setBalance(accountFrom.getBalance() - amount);
```

Comment:
what about negative balance? Try to check it

Reply:
I ckeck it in the setter